### ISSUE TITLE
[BITV] 9.3.3.1/10.1 - New file/folder menu - Incorrectly filled (empty) fields are only highlighted in red. A meaningful error message is missing, e.g. "This field cannot be empty". (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#37081]
- [ ] Fix
- [ ] Backport

<img width="418" alt="17097793f527a5e3340f4ea718df8196" src="https://user-images.githubusercontent.com/1315170/223558910-7c987f06-91e4-4f9d-8003-8c79a4bcba2b.png">

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-c3b56e67ab-v10-n1

</details>

**Comment by skjnldsv** [comment]
Fixed already with F2V on 28
